leadership.html

```diff
@@ -20,65 +20,78 @@
+              <p>                
+                (0) <a href="https://somosazucar.org/">https://somosazucar.org/</a><br>
+                (1) <a href="https://www.laboratoria.la/en">https://www.laboratoria.la/en</a><br>
+                (2) <a href="https://super-8.educa.juegos/">https://super-8.educa.juegos/</a> and <a href="https://jappy.educa.juegos/">https://jappy.educa.juegos/</a><br>
+                (3) <a href="https://repurpose-it.org/?technico_team=sebastian-silva">https://repurpose-it.org/?technico_team=sebastian-silva</a><br>
+              </p>
             </div>
           </div>
           <div class="col-md-4">
             <div class="border-box boxDownload"  >
-              <h3 class="customHeaderStyle">Walter Bender</h3>
+              <h3 class="customHeaderStyle">Sumit Srivastava</h3>
               <figure>
                 <img src="{{ site.baseurl }}/assets/favicon_05.png" class="span1 customLogoImageStyle" alt="Raspberry Pi logo">
               </figure>
-              <p>Walter Bender has more than thirty years experience as an academic advisor at MIT. He has more than 40 years experience as a software developer.</p>
+              <p>Sumit Srivastava is a recipient of [Stability AI supercomputer grant for open source AI research][0] and is currently working on a new AI project called [Cotyper][1]. Prior to this, he started a leading [medical search engine][2] and was a part of the leading talent investor [Entrepreneur First][3]. He is a maintainer of the [Music Blocks programming language][4]. Sumit started many learning groups in Asia where people learn by doing and get iterative feedback. For fun, he plays ukulele and flute, and he builds things like livestreaming software and air purifiers.</p>
+              <p>
+                [0]: <a href="https://hpc.stability.ai/">https://hpc.stability.ai/</a><br>
+                [1]: <a href="https://cotyper.com/">https://cotyper.com/</a><br>
+                [2]: <a href="https://web.archive.org/web/20220123034418/http://coveri.ga/">https://web.archive.org/web/20220123034418/http://coveri.ga/</a><br>
+                [3]: <a href="https://joinef.com/">https://joinef.com/</a><br>
+                [4]: <a href="https://www.sugarlabs.org/music-blocks/">https://www.sugarlabs.org/music-blocks/</a><br>
+              </p>
```

Comment:
Please update formatting to avoid "spillage".

See:
![Screenshot from 2025-01-29 16-47-54](https://github.com/user-attachments/assets/f0c647a3-b1fd-451d-a415-b718656ebfe6)